benchmarks/README.md

```diff
@@ -246,25 +246,8 @@ You can enable `mimalloc` or `snmalloc` (to use either the mimalloc or snmalloc
 cargo run --release --features "mimalloc" --bin tpch -- benchmark datafusion --iterations 3 --path ./data --format tbl --query 1 --batch-size 4096
 ```
 
-The benchmark program also supports CSV and Parquet input file formats and a utility is provided to convert from `tbl`
```

Comment:
the data is generated directly using tpchgen-cli now, no need to convert data with another pass